src/components/FloatingDropdown/FloatingDropdownBase.st.css

```diff
@@ -0,0 +1,91 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color;
+}
+
+:import {
+    -st-from: "../Text/Text.st.css";
+    -st-default: TPAText;
+}
+
+:import {
+    -st-from: "../TextButton/TextButton.st.css";
+    -st-default: TPAButton;
+}
+
+:vars {
+    TextColor: color-5;
+    DisabledTextColor: color-3;
+}
+
+:vars {
+    disabledTextColor: color(value(DisabledTextColor));
+    iconWith: 24px;
+    iconHeight: 24px;
+    maxWidth: 272px;
+    minWidth: 220px;
+    textColor: color(value(TextColor));
+}
+
+.root {
+    -st-extends: TPAButton;
+    display: flex;
+    cursor: pointer;
+    box-sizing: border-box;
+    outline: none;
+    transition: none;
+}
+
+.textContent {
+    max-width: calc(value(maxWidth) - value(iconWith));
+    max-height: value(iconHeight);
+    overflow: hidden;
+    display: flex;
+}
+
+.root .content {
+    -st-mixin: TPAText(
+        MainTextFont "{theme: 'Body-M', size: '16px'}"
+    );
+    display: flex;
+    color: value(textColor);
+}
+
+.arrowIcon path {
+    fill: value(textColor);
+}
+
+.label {
+    display: flex;
+    opacity: 0.6;
+    white-space: nowrap;
```

Comment:
Are we sure about this?
If so, we might need an ellipsis, since we have a max-width, no?

Reply:
We do not want to wrap contend. Not sure about ellipsis. Should we?

Adding: https://github.com/wix/wix-ui-tpa/pull/313/commits/f55adeccf2b747e6e920aaa67369e2d64ea1c1f5

Reply:
From what I see in the spec, we should have the content wrapped:
![image](https://user-images.githubusercontent.com/3277080/79459937-bf86f380-7ffc-11ea-87b2-507d5c04120d.png)
So I guess it would be best to revert these last commits

Reply:
Wait, I made a mistake, I was talking about the option.
And your commit was about the label

Reply:
Note that I modified DropdownBase not dropdown popup.

Reply:
We should check with @michalvardi to see what she says about this

Reply:
![image](https://user-images.githubusercontent.com/1897702/79460240-31f7d380-7ffd-11ea-85e1-46ac583eafa4.png)

Reply:
I was making sure this scenario works fine ^^

Reply:
Yeah, I realised it late 😁
But we should still check with @michalvardi .
I'm guessing we'll go with ellipsis, but we need to be sure.

Reply:
Yes, I asked her :)